Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -1852,8 +1865,8 @@ public void getShortUrl(String alias, String channel, String feature, String sta
      * @see BranchLinkData#putStage(String)
      * @see BranchLinkData#putParams(String)
      * @see BranchLinkCreateListener
-     *
-     * @deprecated use {@link BranchShortLinkBuilder} instead.
+     * @deprecated Use {@link BranchUniversalObject } to create links to your content.
+     * For more details on content sharing please visit <a href="https://gist.github.com/sojanpr/158485479a1bfc0aa7e5">BranchUniversalObject</a>
      */
     @Deprecated
```

Comment:
Hmm ok - I think I need to go in and update all of the documentation. I didn't know we made these deprecated back then.